src/accel/cuda/GPUSolver.h

```diff
@@ -123,23 +123,23 @@ class GPUSolver : public Solver {
    * @return the number of threads per block
    */
   int getNumThreadsPerBlock();
-  FP_PRECISION getFSRSource(int fsr_id, int group);
-  FP_PRECISION getFlux(int fsr_id, int group);
-  void getFluxes(FP_PRECISION* out_fluxes, int num_fluxes);
+  double getFSRSource(long fsr_id, int group) override;
+  double getFlux(long fsr_id, int group) override;
```

Comment:
any technical reason you're switching to double precision ?
It doesnt make much of a difference here

Reply:
Yes, actually. double now has to be used here. Because GPUSolver inherits from Solver, these have to be returning doubles, since that what Solver.h has specified. Otherwise, they won't override their parent class's methods. I had an issue with these not overriding at first, which is why I added the override keywords to give errors in case anyone changes the return value and unknowingly starts making GPUSolver call methods from Solver, which would make it not work.

Reply:
Ok works for me! We may want to change all the solvers to return FP_PRECISION since OpenMOC should be run in single precision most of the time.